### PR TITLE
get_disk(name, id) now alerts users if there are multiple namesake disks

### PR DESCRIPTION
**Pull request by rocknes**
Testing done :

(vcd-cli) C:\code\vcd-cli>vcd disk list
id                                    name    size        size_bytes  status
------------------------------------  ------  --------  ------------  --------
477fe6a5-1fc8-4bf9-b54b-2a7a7849b775  disk1   20 bytes            20  Resolved
b714176a-9c72-48df-9017-c73550e7bee0  disk1   10 bytes            10  Resolved

(vcd-cli) C:\code\vcd-cli>vcd disk delete disk1
Are you sure you want to delete the disk? [y/N]: y
Usage: vcd disk delete [OPTIONS] <name>

Error: Found multiple disks with name disk1, please specify disk id along with disk name.

(vcd-cli) C:\code\vcd-cli>vcd disk delete disk11234
Are you sure you want to delete the disk? [y/N]: y
Usage: vcd disk delete [OPTIONS] <name>

Error: No disk found with the given name/id.

(vcd-cli) C:\code\vcd-cli>vcd disk delete disk1 -i 477fe6a5-1fc8-4bf9-b54b-2a7a7849b775
Are you sure you want to delete the disk? [y/N]: y
task: d3d8edb9-1efe-4b93-b1a8-0acaec563431, Deleted Disk disk1(477fe6a5-1fc8-4bf9-b54b-2a7a7849b775), result: success

### CODE REVIEW

File: pyvcloud/vcd/vdc.py [hunk 1]
```diff
@@ -484,18 +484,29 @@ def get_disk(self, name, disk_id=None):
...
+                    # disk-id's are unique so it's ok to break the loop
+                    # and stop looking further.
+                    break
+        elif name is not None:
+            for disk in disks:
+                if disk.get('name') == name:
+                    if result is None:
+                        result = disk
+                    else:
+                        raise Exception('Found multiple disks with name %s'
```
Comment (sahithi) [inline pyvcloud/vcd/vdc.py]: Did you make sure all the existing commands (vcd-cli) invoking get_disk(..) have an optional parameter to pass the disk_id?
ex: attachdisk, detach disk, update disk etc.

If not, please fix them as part of this change.

Comment (rocknes) [reply pyvcloud/vcd/vdc.py]: Yes, all of them has an optional param 'id'

Comment (rocknes) [reply pyvcloud/vcd/vdc.py]: Wait, update, change owner and delete disk has the optional field id. Attach and detach don't have it.  I will add them. Btw the changes will be in vcd-cli and there won't be any further changes in this pull request.

Comment (sahithi) [reply pyvcloud/vcd/vdc.py]: okay, approving this change, will expect review on vcd-cli

**Pull request merged**